plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -308,22 +310,18 @@ open class DefaultPageCreator(
     protected open fun contentForComments(
         d: Documentable
     ): List<ContentNode> {
-        val tags: GroupedTags = d.documentation.flatMap { (pd, doc) ->
-            doc.children.asSequence().map { pd to it }.toList()
-        }.groupBy { it.second::class }
-
```

Comment:
I don't get this change. Now you are evaluating `groupedTags` every time you access it, so 7 times inside of this function. This is bad.

Reply:
Uh ok, my bad
changed to `val tags = d.groupedTags`